contracts/LoanManager.sol

```diff
@@ -304,7 +304,7 @@ contract LoanManager is Restricted, TokenReceiver {
         return minCollateralRatio > 0 && marginCallRate > 0 && currentRate < marginCallRate;
     }
 
-    function isDefaulted(LoanData storage loan, uint currentRate)
+    function isCollectable(LoanData storage loan, uint currentRate)
```

Comment:
I think it's misleading if we return true for `isCollectable` for loans which are already collected. We should always return `false` if it's already collected.
 AND add an additional flag(s?) to indicate if it was collected because it wasn't repaid on time or it was collected because it went under margin.

Reply:
good point on checking for `Open` status (`Collected` and `Repaid` are not "collectable" indeed) - i've added that to `isCollectable`

about the additional flag: i'm not sure is that is really useful. needs extra storage on blockchain if separate flag. or we could extend the status enum to have 2 kinds of `Collected`. also what if it is both expired and under margin, how to decide what was the "real reason" then?